src/main.cpp

```diff
@@ -57,9 +64,22 @@ int main(int argc, char* argv[]) {
   rectArgs.orderInLayer = 1;
   rectArgs.startingRotation = 0.0f;
 
+<<<<<<< 932d0f710e6a8972c7f0cb87149a34108874135e
```

Comment:
Unresolved merge conflict?

Reply:
Oof.. I have not re-commited my modified main.cpp yet, but that is definitely not a part of the local copy I have. I'll fix this.